src/main/scala/util/GeneratorUtils.scala

```diff
@@ -147,3 +153,24 @@ object ElaborationArtefacts {
     files.foldLeft(false)((t, s) => {s._1 == extension | t})
   }
 }
+
+
+object GenRegDescJson {
```

Comment:
This doesn't really have anything to do with `Generators` i.e. Scala applications that execute Chisel programs.

I suggest moving it to `util/Annotations.scala`